proto/server/server.cc

```diff
@@ -0,0 +1,26 @@
+#include "server.h"
+
+#include <string>
+#include <iostream>
+
+#include <grpc/grpc.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+#include <grpcpp/security/server_credentials.h>
+
+
+/* https://grpc.io/docs/languages/cpp/basics/
+ * Runs server using server builder
+ */ 
+void RunServer(std::string host, std::string port) {
```

Comment:
Actually, I think this should be the main function and both `host` and `port` should be passed from flags.

Reply:
I think it might become a bit too messy to have RunServer and the Main function be the same function so I still kept them separate, LMK if you disagree!